ShutterSystem/shutter.c

```diff
@@ -30,63 +30,86 @@
  #define TMP36		1
  #define LDR		2
  #define SHUTTER	3
-
+ 
+ // temperature & light readings
  float temperatures[MAX_TMP_READINGS];
  uint16_t lightvalues[MAX_LDR_READINGS];
-
+ 
+ 
  int t = 0;
  int l = 0;
-
+ 
+ // set shutter state to up
  enum state shutter_state = UP;
-
+ 
+ // set the leds to a pin
  void initShutter() {
 	outputPin(LEDRED);
 	outputPin(LEDGREEN);
 	outputPin(LEDYELLOW);
-	
+	// green led is standard high, because shutter is up by default
 	setPin(LEDGREEN, HIGH);
  }
-
+ 
+ // reading & storing temperature
  void readTemperature() {
 	uint16_t reading = readADC(TMP_PIN);
-
+	// 5v divided by 1024
 	float voltage = (reading * 5.0) / 1024.0;
+	// setting temperature in Celsius in variable temperature
 	float temperature = (voltage - 0.5) * 100;
+	// adding temperature to array temperatures[t]
```

Comment:
Dit is op zich al vrij duidelijk wat het doet. Als je het wel nodig vind, mag het blijven staan hoor ;)